tests/testlogout.js

```diff
@@ -0,0 +1,31 @@
+import LoginPage from '../pages/login.page';
+import HomePage from '../pages/home.page';
+import LogoutPage from '../pages/logout.page';
+//import { expect } from 'chai';
+
+describe ('TestLogout', async () => {
+
+    it('Should login and logout successfully', async () => {
+
+        await LoginPage.open('login');
+        await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
```

Comment:
You need to login for both your tests in this suite, that means this is duplicated logic. Consider using a **breforeEach()** hook to implement this prerequisite (login with correct credentials). This way you only define these two lines once, and they execute at the begining of every test in this suite.

Eg. 

```javascript
describe('TestLogout', () => {
  beforeEach(async () => {
    await LoginPage.open("login");
    await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
  });

});


```